src/Microsoft.PowerShell.ConsoleGuiTools/OutConsoleGridviewCmdletCommand.cs

```diff
@@ -66,6 +66,16 @@ public SwitchParameter PassThru
         // This method gets called once for each cmdlet in the pipeline when the pipeline starts executing
         protected override void BeginProcessing()
         {
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                ErrorRecord error = new ErrorRecord(
+                    new FormatException("This cmdlet is not supported on this platform."),
```

Comment:
This isn't supported on Windows? It seemed to be working for me the other night when I was playing with it...

Unless it's that it's not fully supported (which is likely as it seemed rather clunky)

Reply:
If you try running ocgv twice it doesn't work

Reply:
https://github.com/migueldeicaza/gui.cs/issues/339